authentication/managing_cloud_provider_credentials/cco-mode-sts.adoc

```diff
@@ -45,8 +45,10 @@ metadata:
   namespace: <target-namespace> <1>
   name: <target-secret-name> <2>
 data:
-  role_name: <operator-role-name> <3>
-  web_identity_token_file: <path-to-token> <4>
+  credentials: |
```

Comment:
for more clarity i would change it from `data` to `stringData` and from
`credentials: |` to
`credentials: |-`
As show in https://github.com/openshift/cloud-credential-operator/blob/master/pkg/cmd/provisioning/aws/create-iam-roles.go#L28

Reply:
ah ok - will do!